django/thunderstore/community/api/experimental/views.py

```diff
@@ -9,23 +11,80 @@
 from thunderstore.community.models import Community
 
 
-class CommunitiesExperimentalApiView(APIView):
-    def get(self, request, format=None):
-        communities = CommunitySerializer(Community.objects.listed(), many=True)
+class CustomCursorPagination(CursorPagination):
+    ordering = "-datetime_created"
+    results_name = "results"
+    page_size = 100
+
+    def get_paginated_response(self, data) -> Response:
         return Response(
-            {
-                "communities": communities.data,
-            },
+            OrderedDict(
+                [
+                    (
+                        "pagination",
+                        OrderedDict(
+                            [
+                                ("next_link", self.get_next_link()),
+                                ("previous_link", self.get_previous_link()),
```

Comment:
I'm not really a fan of returning a link instead of just a new cursor value but there doesn't look like there's a nice way of doing that

Reply:
Link is the REST way